### ISSUE TITLE
Security fence (Heavy CPU load and possible memory leak.)

### ISSUE THREAD

**Issue by Maokuma** [iLexiconn/JurassiCraft#171]
When I  place a "square 4 pole security fence", that I then turn on. My dedicated server goes from idle usage to this: http://s9.postimg.org/3l2k604hr/ram_usage_security_fence.png
One core gets maxed for a few minutes or longer depending on the usage of the in-game fence. And the ram usage goes from idle 100-400mb to "weirdly more". Resulting in CPU performance lag. (It's a Intel dual core.)
The CPU usage goes down after a few minutes of no interaction in the game and the ram usage does seemingly not go down.
Using forge 1291 with McMyAdmin and JurassiCraft-1.3.2-build-103-universal.jar with NEI.
Ubuntu server with java 8 update 31 (64bit)
//I like the Jurassic Park feel with the fence, cool!

**Comment by iLexiconn** [comment]
Is this only serverside?

**Comment by Maokuma** [comment]
It seems the fence performance bug is fixed on server-side regarding fences, I have been running it for hours without any sign's of CPU madness.
I used this to test it:
Forge 1291
JurassiCraft-1.3.2-build-130-universal
CodeChickenCore-1.7.10-1.0.4.35-universal
fastcraft-1.21
NotEnoughItems-1.7.10-1.0.4.95-universal
I don't know what did it in the first place, but placement, size, impossible builds and trowing a waterfall of dinos into it did not make the CPU scream at all. I also tested it on my PC only with the same result. Server tick and the renderer rates was impressively good, even without fastcraft.
I apologize for not testing this sooner and thanks for looking into it!
